data/shipdict.py

```diff
@@ -159,7 +173,7 @@ def add_chunk(self, chunk):
         based on the result of is_abbreviated(),
         gets sent to add_extended or add_abbreviated
         """
-        if self.is_abbreviated(chunk):
```

Comment:
juste un truc pour chipoter, mais on n'a pas vraiment besoin de changer l'appel à la méthode statique, on pourrait peut-être mettre un commentaire dans ce sens pour dire que les deux formes sont valides..

Reply:
Quelque chose comme ceci ?

```python
# Here, we could invoke is_abbreviated on self, it would be still valid.
# Calling ShipDict.is_abbreviated allows to insist that this method is not linked to the instance.
```

Reply:
oui, ou bien 
# here we retrieve the static method through the class
# this form outlines the fact that we're calling a static method
# note that 
# self.is_abbreviated(chunk)
# would work fine just as well